op-deployer/pkg/deployer/pipeline/opchain.go

```diff
@@ -171,27 +180,54 @@ func DeployOPChainGenesisStrategy(env *Env, intent *state.Intent, st *state.Stat
-		Proposer:                thisIntent.Roles.Proposer,
-		Challenger:              thisIntent.Roles.Challenger,
-		BasefeeScalar:           1368,
-		BlobBaseFeeScalar:       801949,
-		L2ChainId:               chainID.Big(),
-		OpcmProxy:               st.ImplementationsDeployment.OpcmProxyAddress,
-		SaltMixer:               st.Create2Salt.String(), // passing through salt generated at state initialization
-		GasLimit:                60_000_000,
-		DisputeGameType:         1, // PERMISSIONED_CANNON Game Type
-		DisputeAbsolutePrestate: common.HexToHash("0x038512e02c4c3f7bdaec27d00edf55b7155e0905301e1a88083e4e0a6764d54c"),
-		DisputeMaxGameDepth:     73,
-		DisputeSplitDepth:       30,
-		DisputeClockExtension:   10800,  // 3 hours (input in seconds)
-		DisputeMaxClockDuration: 302400, // 3.5 days (input in seconds)
+type ChainProofParams struct {
+	DisputeGameType                         uint32      `json:"disputeGameType" toml:"disputeGameType"`
+	DisputeAbsolutePrestate                 common.Hash `json:"disputeAbsolutePrestate" toml:"disputeAbsolutePrestate"`
+	DisputeMaxGameDepth                     uint64      `json:"disputeMaxGameDepth" toml:"disputeMaxGameDepth"`
+	DisputeSplitDepth                       uint64      `json:"disputeSplitDepth" toml:"disputeSplitDepth"`
+	DisputeClockExtension                   uint64      `json:"disputeClockExtension" toml:"disputeClockExtension"`
+	DisputeMaxClockDuration                 uint64      `json:"disputeMaxClockDuration" toml:"disputeMaxClockDuration"`
+	DangerouslyAllowCustomDisputeParameters bool        `json:"dangerouslyAllowCustomDisputeParameters" toml:"dangerouslyAllowCustomDisputeParameters"`
+}
+
+func makeDCI(intent *state.Intent, thisIntent *state.ChainIntent, chainID common.Hash, st *state.State) (opcm.DeployOPChainInput, error) {
+	proofParams, err := jsonutil.MergeJSON(
+		ChainProofParams{
+			DisputeGameType:         standard.DisputeGameType,
+			DisputeAbsolutePrestate: standard.DisputeAbsolutePrestate,
+			DisputeMaxGameDepth:     standard.DisputeMaxGameDepth,
+			DisputeSplitDepth:       standard.DisputeSplitDepth,
+			DisputeClockExtension:   standard.DisputeClockExtension,
+			DisputeMaxClockDuration: standard.DisputeMaxClockDuration,
+		},
+		intent.GlobalDeployOverrides,
+		thisIntent.DeployOverrides,
+	)
+	if err != nil {
+		return opcm.DeployOPChainInput{}, fmt.Errorf("error merging proof params from overrides: %w", err)
 	}
+
+	return opcm.DeployOPChainInput{
+		OpChainProxyAdminOwner:       thisIntent.Roles.L1ProxyAdminOwner,
+		SystemConfigOwner:            thisIntent.Roles.SystemConfigOwner,
+		Batcher:                      thisIntent.Roles.Batcher,
+		UnsafeBlockSigner:            thisIntent.Roles.UnsafeBlockSigner,
+		Proposer:                     thisIntent.Roles.Proposer,
+		Challenger:                   thisIntent.Roles.Challenger,
+		BasefeeScalar:                1368,
+		BlobBaseFeeScalar:            801949,
```

Comment:
Where do these values come from? Could they be moved into a standards or defaults file?

Reply:
I can move them into the standard file.